client/webserver/live_test.go

```diff
@@ -443,18 +560,49 @@ func (c *TCore) Cancel(pw string, sid string) error {
 	return nil
 }
 
-func (c *TCore) NotificationFeed() <-chan core.Notification { return make(chan core.Notification, 1) }
+func (c *TCore) NotificationFeed() <-chan core.Notification { return c.noteFeed }
+
+func (c *TCore) runEpochs() {
+	epochTick := time.NewTimer(time.Second).C
+out:
+	for {
+		select{
+		case <-epochTick:
+			epochTick = time.NewTimer(epochDuration - time.Since(time.Now().Truncate(epochDuration))).C
+			c.noteFeed <- &core.EpochNotification{
+				Notification: db.NewNotification("epoch", "", "", db.Data),
+				Epoch:        getEpoch(),
+			}
+		case <-tCtx.Done():
+			break out
+		}
+	}
+}
 
 func TestServer(t *testing.T) {
-	ctx, shutdown := context.WithCancel(context.Background())
+	numBuys = 0
+	numSells = 0
+	feedPeriod = 5000 * time.Millisecond
+	register := true
```

Comment:
Just a reminder to `go test -tags live -timeout 60m` and connect to `localhost:54321` for the test server. I've added `numBuys`, `numSells`, and `feedPeriod` to control the initial state of the orderbook, and `register` to control the initial user registration state on the test server so you don't have to go through all of the forms. Try the live server with `feedPeriod = 250 * time.Millisecond` too.